docs/options.rst

```diff
@@ -19,15 +19,29 @@ Also, options can be set and inspected using the respective commands of the inpu
   :py:func:`getOptionNames() <cvc5.pythonic.Solver.getOptionNames()>`,
   :py:func:`getOptionInfo() <cvc5.pythonic.Solver.getOptionInfo()>`
 
-Generally, all options are identified by a name, and they can additionally have one or more aliases (that can be used instead of their name anywhere) and a short name (a single letter).
-Internally, options are strongly typed and must be either Boolean, (signed or unsigned) integers, floating-point numbers, strings, or an enumeration type. Values for options with a numeric type may be restricted to an interval. A few options have custom types (for example :ref:`err <lbl-option-err>`) that require special treatment internally.
+Generally, all options are identified by a name ``<name>``, and a short name
+``<short>`` (a single letter).
```

Comment:
To me, this suggests that every option has a short name (which is not the case).